test/workflows/components/mnist.jsonnet

```diff
@@ -20,7 +20,7 @@ local defaultParams = {
   // Which Kubeflow cluster to use for running TFJobs on.
   kfProject: "kubeflow-ci-deployment",
   kfZone: "us-east1-b",
-  kfCluster: "kf-vmaster-n01",
+  kfCluster: "kf-v0-6-n00",
```

Comment:
This is causing the tests to run against a v0-6 cluster. Do we really want to do that?
Don't we want to test against a cluster that's running master?

Reply:
Making the test pass doesn't help us if the test doesn't test what we care about. The meaning of the dashboard for periodic master should mean the tests are passing on clusters deployed on master.

So if we actually run on v0-6 clusters then we are creating misleading signal.

There are two possible paths forward

1. Fix the test so it passes on master (ideal) 
1. Mark the test as expected to fail so that the dashboard correctly reflects that the test is failing on master but marking it as expected to fail doesn't block presubmits

One way to achieve that is to convert the test to use pytest. Then we can just annotate it to indicate its expected to fail.

We will need to update the workflow to invoke it using pytest.

You could either update the existing jsonnet file or wait for #666 and then add it to the pyfunc which might be easier then modifying ksonnet.

Reply:
OK, Let's mark the test as expected to fail, and update to use pytest later :-)